src/aspect.ts

```diff
@@ -14,7 +16,7 @@ export interface ECRRepositoryAspectProps {
    *
    * @default - process.env.BSS_IMAGE_ASSET_ACCOUNT_ID
    */
-  readonly imageAssetAccountId?: string;
+  readonly imageAssetAccountId ? : string;
```

Comment:
I prefer not to add a space in between.
please use `readonly imageAssetAccountId ? : string;`

Reply:
Looks like it's changed by `eslint`, something is introduced in recent update?

Reply:
Let me check

Reply:
@wchaws any finding?

Reply:
@zxkane I've checked in gitpod env. eslint won't add space.
And eslint allow this kind of syntax. I guest it's caused by editor. 
Again, I still prefer to manually eliminate those spaces.